pymeasure/instruments/rohdeschwarz/sfm.py

```diff
@@ -0,0 +1,1501 @@
+        """,
+        validator = strict_discrete_set,
+        values=["BG_G", "DK_G", "I_G", "L_G", "GERM","BELG", "NETH",
+        "FIN", "AUST", "BG_T", "DENM", "NORW", "SWED", "GUS", "POL1", "POL2",
+        "HUNG", "CHEC", "CHINA1", "CHINA2" , "GRE", "SAFR", "FRAN", "USA",
+        "KOR" , "JAP", "CAN", "SAM"],
+            )
+
+    #output voltage (3.6.6.12)
+    #TODO  LEV-->LEV[1..6]
+    output_voltage = Instrument.control(
+        "SOUR:VOLT:LEV?",
+        "SOUR:VOLT:LEV %g",
+        """A float property controlling the output level in Volt,
+
+        Minimum 2.50891e-6, Maximum 0.707068 (depending on output mode)
+        """,
+        validator = strict_range,
+        values = [2.508910e-6,0.7070168],
+        )
+
+    #STATUS entries (3.6.7)
+    event_reg = Instrument.measurement(
+        "STAT:OPER:EVEN?",
+        """
+        Content of the event register of the Status Operation Register
+        """,
+        cast=int,
+        )
+
+    status_reg = Instrument.measurement(
+        "STAT:OPER:COND?",
+        """
+        Content of the condition register of the Status Operation Register
+        """,
+        cast=int,
+        )
+
+    operation_enable_reg = Instrument.control(
+        "STAT:OPER:ENAB?",
+        "STAT:OPER:ENAB %d",
+        """
+        Content of the enable register of the Status Operation Register
+
+        Valid range: 0...32767
+        """,
+        cast=int,
+        validator = strict_range,
+        values=[0, 32767]
+        )
+
+    def status_preset(self):
+        """ partly resets the SCPI status reporting structures
+
+        """
+        self.write("STAT:PRES")
+
+    questionable_event_reg = Instrument.measurement(
+        "STAT:QUES:EVEN?",
+        """
+        Content of the event register of the Status Questionable Operation Register
+        """,
+        cast=int,
+        )
+
+    questionanble_status_reg = Instrument.measurement(
+        "STAT:QUES:COND?",
+        """
+        Content of the condition register of the Status Questionable Operation Register
+        """,
+        cast=int,
+        )
+
+    questionable_operation_enable_reg = Instrument.control(
+        "STAT:QUES:ENAB?",
+        "STAT:QUES:ENAB %d",
+        """
+        Content of the enable register of the Status Questionable Operation Register
+
+        Valid range 0...32767
+        """,
+        cast=int,
+        validator = strict_range,
+        values=[0, 32767]
+        )
+
+    #SYSTEM entries (3.6.8)
+    beeper = Instrument.control(
+        "SYST:BEEP:STATE?",
+        "SYST:BEEP:STATE %s",
+        """ A bool property that controls the beeper status,
+
+        ======  =======
+        Value   Meaning
+        ======  =======
+        False   beeper disabled
+        True    beeper enabled
+        ======  =======
+        """,
+        validator = strict_discrete_set,
+        values={False:0, True:1},
+        map_values = True,
+        )
+
+    display_update = Instrument.control(
```

Comment:
status_info_shown?

Reply:
Thanks that is a better name for this property